magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala

```diff
@@ -27,7 +28,12 @@ case class UpdateCloudFormationTask(cloudFormationStackName: String, template: P
     MessageBroker.info(s"Parameters: $actualParameters")
 
     if (CloudFormation.describeStack(cloudFormationStackName).isDefined)
-      CloudFormation.updateStack(cloudFormationStackName, template.string, actualParameters)
+      try {
+        CloudFormation.updateStack(cloudFormationStackName, template.string, actualParameters)
+      } catch {
+        case ase:AmazonServiceException if ase.getMessage contains "No updates are to be performed." =>
```

Comment:
I was thinking Amazon error code, which there seems to be a method for on `AmazonServiceException`